rust/windows-client/src-tauri/src/client/gui.rs

```diff
@@ -545,7 +554,7 @@ impl From<connlib_client_shared::ResourceDescription> for ResourceDisplay {
             connlib_client_shared::ResourceDescription::Cidr(x) => Self {
                 id: x.id,
                 name: x.name,
-                // TODO: CIDRs aren't URLs right?
+                // // TODO: CIDRs aren't URLs right?
```

Comment:
```suggestion
                // TODO: CIDRs aren't URLs right?
```
😛 

Also, we could add methods to retrieve these directly from the `ResourceDescription` so we don't have to define this new type.

Reply:
True. I'll make a TODO to add those methods, #2961 might have conflicts if I rename it now, because I moved this code around in there.